EzSystemsPrivacyCookieBundle.php

```diff
@@ -5,8 +5,13 @@
  */
 namespace EzSystems\PrivacyCookieBundle;
 
+use EzSystems\PrivacyCookieBundle\DependencyInjection\EzSystemsPrivacyCookieExtension;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 class EzSystemsPrivacyCookieBundle extends Bundle
 {
+    public function getContainerExtension()
+    {
+        return new EzSystemsPrivacyCookieExtension();
```

Comment:
No, it's not. If I remove it I get:

```
Users will expect the alias of the default extension of a bundle to be the underscored version of the bundle name ("ez_systems_privacy_cookie"). You can override "Bundle::getContainerExtension()" if you want to use "ez_privacy_cookie" or another alias.
```

if I then remove https://github.com/ezsystems/EzSystemsPrivacyCookieBundle/pull/12/files#diff-4a17c16aafd5033adf08c1830f584d2aR29 I will not be able to load configuration within `config.yml` under `ez_privacy_cookie` key (ok, `ez_systems_privacy_cookie` will work, but it's less sexy :smiling_imp: ).